episcalp/scripts/bids/run_bids_conversion_upmc.py

```diff
@@ -0,0 +1,165 @@
+"""Run this file for converting Jefferson scalp EEG data."""
+from pathlib import Path
+import pandas as pd
+import re
+
+from mne_bids.path import BIDSPath
+
+from episcalp.bids.bids_conversion import (
+    write_epitrack_bids,
+    append_original_fname_to_scans,
+)
+from episcalp.bids.utils import update_participants_info
+
+
+def convert_upmc_to_bids():
+    root = Path("D:/OneDriveParent/Johns Hopkins/UPMC_Scalp - Documents/scalp_study/root")
+    source_root = root / "sourcedata"
+
+    # epilepsy source data from JHH is organized into three sub-folders
```

Comment:
```suggestion
    # epilepsy source data from UPMC is organized into three sub-folders
```

Are we getting all three categories?